.idea/workspace.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do you think it is worth sharing?

Reply:
lol, whoops.

Reply:
Hadn't added it to my global gitignore yet.